.github/workflows/TestEnv.yml

```diff
@@ -39,7 +40,10 @@ jobs:
           ddev composer config --no-plugins allow-plugins.lullabot/drainpipe true
           ddev composer config repositories.drainpipe --json "{\"type\": \"path\", \"url\": \"drainpipe\", \"options\": {\"symlink\": false}}"
           ddev composer config minimum-stability dev
-          cat composer.json | jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' | tee composer.json > /dev/null
+          # cat + tee still has a race condition causing random failures.
+          # https://stackoverflow.com/a/68676400
+          cat composer.json | jq --indent 4 '."autoload-dev" = {"files": ["vendor/lullabot/drainpipe/scaffold/env/dotenv.php"]}' | tee composer.new.json
+          mv composer.new.json composer.json
```

Comment:
won't you need `mv -f` if you are overwriting the file?

Reply:
Nope!

```
andrew@drupal-10-composer-web:~$ touch test
andrew@drupal-10-composer-web:~$ touch test2
andrew@drupal-10-composer-web:~$ mv test test2
andrew@drupal-10-composer-web:~$ ls
README.txt  bash_aliases.example  test2
```

Reply:
Strange, I've definitely encountered that before.

Reply:
That happens if the file doesn't have `+w`:

```
andrew@drupal-10-composer-web:~$ touch test test2; chmod a-w test2; mv -v test test2
mv: replace 'test2', overriding mode 0444 (r--r--r--)?
```

Reply:
Might have been a mutagen thing. I remember now that I needed it here, but chose to just rm the file first instead: https://github.com/TugboatQA/images/blob/main/scripts/bakefile.sh#L41